### PR TITLE
Final Kettering changes

### PR DESCRIPTION
**Pull request by scarmain**
Ran these changes on Tuesday and plan to use them for Kettering Saturday.

### CODE REVIEW

File: src/main/java/frc/robot/ShooterAuton.java [hunk 1]
```diff
@@ -15,10 +15,10 @@
     private WPI_TalonSRX hopper;
     private WPI_TalonSRX ejector;
     private ArrayList<ShotChoice> choices;
-
+    
     public ShooterAuton() {
         holo = HoloTable.getInstance();
-        driveTrain = new DriveTrain();
+        driveTrain = Robot.driveTrain;
```
Comment (programmer94) [inline src/main/java/frc/robot/ShooterAuton.java]: Any chance you can make driveTrain a private variable with a public accessor? Thanks!

File: src/main/java/frc/robot/command/AutoTurn.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    private PigeonIMU gyro;
+    private HoloTable table;
+    private DriveTrain driveTrain;
+    private double commandAngle, startAngle, currentAngle;
+    private boolean isFinished;
+
+    public AutoTurn(double commandAngle) {
+        table = HoloTable.getInstance();
+        gyro = table.getGyro();
+        driveTrain = Robot.driveTrain;
```
Comment (programmer94) [inline src/main/java/frc/robot/command/AutoTurn.java]: I would prefer driveTrain to be a private variable.

File: src/main/java/frc/robot/ShooterAuton.java (hunk 1 above)
Comment (scarmain) [reply src/main/java/frc/robot/ShooterAuton.java]: Personally, I hate public accessors, variables can just be public, but I get it is Java style.  

For this year, I'm actually going to push for a different design strategy (6861 code did this), where the constructor requires all the subsystems it is going to modify (so like the Auton class would have constructor parameters of gyro, drivetrain, etc).  This way, the classes are much easier to test, as we can inject dummy classes in of the same interface to manipulate during testing.

I am drafting this change though for a public accessor.

Comment (programmer94) [reply src/main/java/frc/robot/ShooterAuton.java]: I know accessors are a pain, but yeah. That's Java. I'm open to new ideas when it comes to design patterns. We can discuss it in more detail at some point.

**Review by programmer94 — APPROVED** [review]
Looks good. Thanks for making those changes.

**Pull request merged**